client/src/components/pages/CreatePostPage/Event/style.css

```diff
@@ -0,0 +1,15 @@
+.error {
```

Comment:
@aminalakhsham You might need to make these classes more specific so that these styles dont' bleed into other components with similar classnames.  For example, you could change to:
`.main--eventForm .error`
`.main--eventForm .ant-upload, .main--eventForm .ant-btn`
and so on

Reply:
thank you for reminding about that class I don't use it (i try to use :laughing: it when I use Foemik with ant )